src/js/Tables/BaseTable.js

```diff
@@ -62,7 +62,17 @@ class BaseTable {
 		this.wrapper = this.rootEl.closest('.o-table-scroll-wrapper');
 		this.container = this.rootEl.closest('.o-table-container');
 		this.overlayWrapper = this.rootEl.closest('.o-table-overlay-wrapper');
+		/**
+		 * @property {Object|Null} _currentSort - The current sort applied.
+		 * @property {Number} _currentSort.columnIndex - The index of the currently sorted column.
+		 * @property {String} _currentSort.sortOrder - The type of sort, "ascending" or "descending"
+		 */
```

Comment:
Woah, does this work with VSCode intellisense and with JSDoc documentation generation?!
Lee, you are too good :-)

Reply:
Thanks! It works with intellisense and I'm hoping it will work with our JSDoc documentation (think it will).